de.tu_bs.cs.isf.e4cf.parts.project_explorer/src/de/tu_bs/cs/isf/e4cf/parts/project_explorer/wizards/NewFileDialog.java

```diff
@@ -0,0 +1,83 @@
+package de.tu_bs.cs.isf.e4cf.parts.project_explorer.wizards;
+
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import de.tu_bs.cs.isf.e4cf.core.stringtable.E4CFileTable;
+import de.tu_bs.cs.isf.e4cf.core.util.services.RCPImageService;
+import de.tu_bs.cs.isf.e4cf.parts.project_explorer.stringtable.FileTable;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.TextField;
+import javafx.scene.control.TextInputDialog;
+import javafx.stage.Stage;
+
+/** A wizard that lets the user create a new file. */

```

Comment:
Dialog is not Wizard
```suggestion
/** A dialog that lets the user create a new file. */
```

Reply:
Right now the dialogs are also inside the wizards package, should i move them to a new one?

Reply:
changed

Reply:
Sure